src/integrationTest/docker/wiremock/__files/users.json

```diff
@@ -0,0 +1,32 @@
+{
+    "users": [
+        {
+            "id": "1",
+            "createdAt": "2025-03-23T12:00:00Z",
+            "projectId": "radar",
+            "userId": "sub-1",
+            "humanReadableUserId": null,
+            "sourceId": "garmin",
+            "externalId": null,
+            "isAuthorized": true,
+            "startDate": "2024-09-23T12:00:00Z",
+            "endDate": "2025-09-23T12:00:00Z",
+            "version": null,
+            "serviceUserId": "sub-1"
+        },
+        {
+            "id": "2",
+            "createdAt": "2025-03-23T12:00:00Z",
+            "projectId": "radar",
+            "userId": "sub-2",
+            "humanReadableUserId": null,
+            "sourceId": "garmin",
+            "externalId": null,
+            "isAuthorized": true,
+            "startDate": "2024-09-23T12:00:00Z",
+            "endDate": "2025-09-23T12:00:00Z",
```

Comment:
Can we have a user where the `endDate` is null? It should throw an error now, but this is currently nullable in rest-source-auth. I think this still needs to be discussed if we want to support this or not.